go.mod

```diff
@@ -1,44 +1,68 @@
 module github.com/moby/vpnkit
 
-go 1.16
+go 1.17
 
 require (
 	github.com/Microsoft/go-winio v0.4.11
+	github.com/google/uuid v0.0.0-20170814143639-7e072fc3a7be
+	github.com/labstack/echo v3.3.10+incompatible
+	github.com/linuxkit/virtsock v0.0.0-20180830132707-8e79449dea07
+	github.com/pkg/errors v0.8.1-0.20181023235946-059132a15dd0
```

Comment:
This version is not compatible with go1.13 error wrapping; we should update that as well